### DISCUSSION TITLE
what should i pass to `Firebase.Database.get_database_reference()`?

### DISCUSSION THREAD

**Discussion by MeBadDev** [GodotNuts/GodotFirebase#322] [Q&A] [answered]
<img width="703" alt="image" src="https://user-images.githubusercontent.com/94725241/205416142-e7c68baa-f0d2-4bc4-a31e-33b1d7881efb.png">
My database look like this, and the wiki says i need `FireBase.Database.get_database_reference()`, and i need to pass `path` to it, what should i pass? url to the database?

**Comment by MeBadDev** [comment]
so now i had logged in, how could i get data in database? patch_data_updated and new_data_updated isn't emitting

**Comment by fenix-hub** [comment, marked as answer]
Once you've properly configured the plugin (set the `databaseUrl` property) you can get a reference to your database like this
```gdscript
Firebase.Database.get_database_reference("quotes") # in your case
```

**Reply by fenix-hub** [reply]
To get a copy of your database at that reference you can do 
```gdscript
Firebase.Database.get_database_reference("quotes").get_data()
```

**Reply by MeBadDev** [reply]
thanks! at first it gave me an error because i didn't wait for login
